tests/unit/models/requests/test_channel_authorize.py

```diff
@@ -56,3 +56,14 @@ def test_has_multiple_signing_methods_is_invalid(self):
                 passphrase=_DUMMY_STRING,
                 seed_hex=_DUMMY_STRING,
             )
+
+    def test_throws_if_positional_args_mixed_with_non_positional_args(self):
```

Comment:
Why is this test here?

Reply:
I wanted a unit test that shows that positional arguments are not permitted in the constructor of ledger objects. that was my intention

Reply:
Updated in [0cbc07e](https://github.com/XRPLF/xrpl-py/pull/673/commits/0cbc07eca9a5e647bf1be57c4e4403c888edf230)
let me know if that looks ok